src/labs/requests/NewLabRequest.tsx

```diff
@@ -72,7 +72,7 @@ const NewLabRequest = () => {
       Toast(
         'success',
         t('states.success'),
-        `${t('lab.successfullyCreated')} ${newLab.type} ${newLab.patient}`,
+        `Successfully created ${newLab.type} for ${newLab.fullName}`,
```

Comment:
May you use the value from the local i.e t('lab.successfullyCreated') rather than hard-coding the text. You may create it if it doesn't already exist in the locale file.

Reply:
I will make this change right now.

Reply:
Thanks, it also seems you have merge conflicts (see at the bottom of this PR). May you address those too.

Reply:
I believe I'm all set now, @blestab. Thank you in advance for your patience with me, this is my first time attempting to contribute to an open-source project.

Reply:
That's alright @gwal21, thank you for helping us with this one. Well done so far, you are almost there!  Looks like we are left with a linter error. To resolve this, in your local development environment run `npm run lint:fix` and that should hopefully resolve the issue

Reply:
@blestab, I have a quick question. Running this `npm run lint:fix` command I see there are 70 warnings, all found on files that I haven't had any interaction with. Am I required to resolve all of these warnings before this PR is able to be implemented?

Reply:
Nop we are concerned about just the 1 error for now

Reply:
I'm going through and resolving the warnings, I will update once I have completed.

Reply:
Hi, thanks, to keep the PR atomic and related directly to the issue it resolves, we can maybe create a separate issue for you to look at those.

Reply:
@gwal21 i have logged a separate issue for the resolution of the warnings, Go ahead and commit a fix for the error only. You can create a separate PR for the warnings

Reply:
@blestab, Sorry I accidentally mixed up and pushed from my work account with a new branch. Therefore you will see another pull request. I believe this one should be all set though. As I'm only showing 4 lint warnings which are relating to the a couple tests.